datafusion/src/scalar.rs

```diff
@@ -86,6 +86,10 @@ pub enum ScalarValue {
     IntervalYearMonth(Option<i32>),
     /// Interval with DayTime unit
     IntervalDayTime(Option<i64>),
+
```

Comment:
nitpick. Delete the blank line to keep the same as above.

Reply:
Done in e25e500